pipeline/metadata/flatten_hyperquack.py

```diff
@@ -186,3 +192,5 @@ def _process_hyperquack_v2(self, filename: str, scan: Any,
         row.error = response['error']
 
       yield row
+      if not is_control:
+        retry_index += 1
```

Comment:
Is this not better to put before the yield?

Reply:
No, this would cause an off-by-one error. The retry indexes start at 0.